src/biotite/structure/bonds.pyx

```diff
@@ -1620,20 +1620,22 @@ def connect_via_residue_names(atoms, bint inter_residue=True,
     """
     from .info.bonds import bonds_in_residue
     from .residues import get_residue_starts
+    from .info.ccd import get_from_ccd
 
     cdef list bonds = []
     cdef int res_i
     cdef int i, j
     cdef int curr_start_i, next_start_i
     cdef np.ndarray atom_names = atoms.atom_name
     cdef np.ndarray atom_names_in_res
+    cdef np.ndarray std_atom_ids
```

Comment:
With the changes below this variable cannot be statically typed anymore as the type changes from `BinaryCIFColumn` to `ndarray`. We are not excessively accessing its methods anyway so there is probably only marginal performance benefit from typing it.